### ISSUE TITLE
Util - Add module to switch unit sides

### ISSUE THREAD

**Issue by astruyk** [astruyk/Ares#126]
From forums ( http://forums.bistudio.com/showthread.php?185254-Ares-Modules-expanding-Zeus-functionality-Release-Thread&p=2812885&viewfull=1#post2812885 ):

> Is is possible to add module that switch unit/player side? So team play as blu, but i want as ZEUS to create coverd mission, so I switch units players to opfor, and when they are in turn them again blue?

**Comment by TheOtherTaylor** [comment]
I stumbled across this:
> _newgroup = createGroup East; [player] join _newgroup;
I'll test it out later.  It is probably easy to adapt this to swap whole groups or whole sides.

**Comment by astruyk** [comment]
Tried using `setFriend` to change the friendlyness of troops - seems to only affect units spawned AFTER the change has been made. For example, if you spawn a NATO group and a CSAT group, and then change NATO and CSAT to be friends they will continue to fight. If you spawn a new group after that, the old group will attack them but the new group will not.

Test code for reference:
```
if (isClass (configFile >> "CfgPatches" >> "Ares")) then
{
	[
		"Util",
		"Change Friendlyness",
		{
			_eastAndWestFriendsDefault = if ((east getFriend west) >= 0.6) then { 0 } else { 1 };
			_eastAndResistanceFriendsDefault = if ((east getFriend resistance) >= 0.6) then { 0 } else { 1 };
			_westAndResistanceFriendsDefault = if ((west getFriend resistance) >= 0.6) then { 0 } else { 1 };
			_dialogResult = [
				"Change Friendlyness",
				[
					["Blufor and Opfor", ["Friends", "Not Friends"], _eastAndWestFriendsDefault],
					["Blufor and Greenfor", ["Friends", "Not Friends"], _eastAndResistanceFriendsDefault],
					["Opfor and Greenfor", ["Friends", "Not Friends"], _westAndResistanceFriendsDefault]
				]
			] call Ares_fnc_ShowChooseDialog;
			
			if (count _dialogResult > 0) then
			{
				_dialogEastAndWestFriends = _dialogResult select 0;
				_dialogEastAndResistanceFriends = _dialogResult select 0;
				_dialogWestAndResistanceFriends = _dialogResult select 0;

				east setFriend [west, if (_dialogEastAndWestFriends == 0) then { 1 } else { 0 }];
				east setFriend [resistance, if (_dialogEastAndResistanceFriends == 0) then { 1 } else { 0 }];
				west setFriend [resistance, if (_dialogWestAndResistanceFriends == 0) then { 1 } else { 0 }];
			};
		}
	] call Ares_fnc_RegisterCustomModule;
};
```

**Comment by astruyk** [comment]
The 'create a group and add the friendlies to it' trick seems to work locally at least...

**Comment by astruyk** [comment]
First pass implemented in 50408ce7271a069dc828700407c5c4aab70c8507
